Gemfile

```diff
@@ -3,4 +3,6 @@ gem 'guard-sass'
 gem 'guard-coffeescript'
 gem 'html2haml'
 gem 'guard-haml'
+gem 'guard-shell'
 gem 'rb-readline'
+gem 'therubyracer'
```

Comment:
Sorry, I should have mentioned: `execjs` is actually what's being used as a runner (and was already pulled in by bundler).  However, with the previous Gemfile it raised errors because it was missing a JavaScript _runtime_.  So, I added `therubyracer` to provide `execjs` with a JavaScript runtime that could be installed using bundler.